src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -4,60 +4,67 @@ metaDescription: "A lightweight tool to monitor your OpenAI workload."
 freshnessValidatedDate: never
 ---
 
-import modelperformancemonitoringOpenAIDashboard from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard.webp'
-
-With the GPT series application integration, users have the ability to seamlessly monitor OpenAI completion queries and simultaneously log useful statistics in a New Relic customizable dashboard.
-
-By adding just two lines of code, users can gain access to key performance metrics such as cost, response time, and sample inputs/outputs. The dashboard also allows users to track total requests, average token/requests, and model names.
-
-With the fully customizable dashboard of New Relic, users can add additional metrics based on their specific business requirements and needs.
-
-Overall, New Relic's GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
+import modelperformancemonitoringOpenAIDashboard1 from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard-1.webp'
 
+With our OpenAI integration, you have the ability to seamlessly monitor all OpenAI queries. By adding just two lines of code, your team can gain access to key performance metrics such as cost, response time, rate limits, error rates, chat history, and sample inputs and outputs. Our OpenAI integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
 
 <img
   title="OpenAI dashboard"
   alt="OpenAI dashboard"
-  src={modelperformancemonitoringOpenAIDashboard}
+  src={modelperformancemonitoringOpenAIDashboard1}
 />
 
+This guide will show you how to integrate your OpenAI applications so you can start monitoring your data.
 
-## Installation [#installation]
+<CollapserGroup>
+  <Collapser 
+    id="install-with-python" 
+    title="Install for Python applications"
+  >
 
-Installation is straightforward and similar to any Python library.
+<Steps>
+<Step>
+### Install OpenAI with pip
 
-```python
+After you've completed your OpenAI import, run: 
+
+```bash
 pip install nr-openai-observability
 ```
+</Step>
+<Step>
+### Set your environment variable [#set-environment-variable]
 
-## Getting Started [#getting-started]
-
-This guide takes you step by step for everything needed to integrate OpenAI GPT applications and start monitoring your usage.
+Get your <InlinePopover type="licenseKey" /> and set it as environment variable: `NEW_RELIC_LICENSE_KEY`. 
```

Comment:
I think you might need a small lead-in here, it feels kind of abrupt to start the section with "Get your license key". Even just a sentence or a start like "The first thing you need to do" or something along those lines would help soften the abruptness.